apps/passport-server/test/genericIssuance.spec.ts

```diff
@@ -175,22 +190,28 @@ describe("Generic Issuance", function () {
         feedFolder: "Edge City",
         feedId: "edge-city"
       },
-      lemonadeApiKey: EdgeCityDenverBouncer.apiKey,
+      oauthAudience: "test",
+      oauthClientId: "edge-city-client-id",
+      oauthClientSecret: "test",
+      oauthServerUrl: "test",
+      backendUrl: lemonadeBackendUrl,
```

Comment:
do these hard-coded oauth values need to match up with values elsewhere in this PR? i.e. does the string literal `edge-city-client-id` need to match up with itself somewhere else? how were these values picked? are they robust to being changed? I think it matches up with `EdgeCityLemonadeAccount`, so `edge-city-client-id` should be extracted to a constant. I am assuming that the remaining oauth fields here whose values are just `"test"` are irrelevant for testing.